README.md

```diff
@@ -46,16 +46,22 @@ If you're not familiar with git, consider learning the basics from one of the ma
 
 In order to start contributing, simply follow these 10 steps:
 
-1. Please make sure you've read the [CONTRIBUTING.md](https://github.com/acs-upb-mobile/acs-upb-mobile/blob/master/CONTRIBUTING.md) file.
+1. Please make sure you've read through the [CONTRIBUTING.md](https://github.com/acs-upb-mobile/acs-upb-mobile/blob/master/CONTRIBUTING.md) file.
 2. Fork the [ACS UPB Mobile](https://github.com/acs-upb-mobile/acs-upb-mobile) repository.
-3. Clone the fork onto your system and open it as an Android Studio project.
+3. Clone the fork onto your system (`git clone`) and open it as an Android Studio project.
```

Comment:
Since it is a workshop, I would suggest maybe giving a little mention that although there may be other code editors or IDEs for coding Flutter, Android Studio would be the recommended solution. Maybe put this as a little paranthesis in as few words as possible